### PR TITLE
Move files to /css/css_backgrounds_and_borders

### LINKED ISSUE #8017: Content bug: The CSS_Background_and_Borders directory is misspelled
## What page(s) did you find the problem on?

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Background_and_Borders/Border-image_generator
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Background_and_Borders/Border-radius_generator
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Background_and_Borders/box-shadow_generator

## Specific page section or heading?

Their URL

## What is the problem?

These pages are in the CSS_Background_and_Borders directory, but it seems a misspell of CSS_Backgrounds_and_Borders.

## What did you expect to see?

They should be in 
/docs/Web/CSS/CSS_Backgrounds_and_Borders/
(which already exists) instead of
/docs/Web/CSS/CSS_Background_and_Borders/

### PR DESCRIPTION
**Pull request by AnilSeervi**
- Fixes #8017
- Follow up of #8044

### CODE REVIEW

File: files/en-us/web/css/css_backgrounds_and_borders/box-shadow_generator/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Box-shadow generator
-slug: Web/CSS/CSS_Background_and_Borders/Box-shadow_generator
+slug: Web/CSS/CSS_Backgrounds_and_Borders/box-shadow_generator
```
Comment (teoli2003) [inline files/en-us/web/css/css_backgrounds_and_borders/box-shadow_generator/index.md:3]: Capitalization is incorrect: Box-shadow-generator

File: files/en-us/web/css/css_backgrounds_and_borders/border-radius_generator/index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Border-radius generator
-slug: Web/CSS/CSS_Background_and_Borders/Border-radius_generator
+slug: Web/CSS/CSS_Backgrounds_and_Borders/border-radius_generator
```
Comment (teoli2003) [inline files/en-us/web/css/css_backgrounds_and_borders/border-radius_generator/index.md:3]: Capitalisation incorrect: Border-radius-generator

File: files/en-us/web/css/css_backgrounds_and_borders/border-image_generator/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Border-image generator
-slug: Web/CSS/CSS_Background_and_Borders/Border-image_generator
+slug: web/css/css_backgrounds_and_borders/border-image_generator
```
Comment (teoli2003) [inline files/en-us/web/css/css_backgrounds_and_borders/border-image_generator/index.md:3]: Capitalization is incorrect: Web/CSS/CSS_Background_and_Borders/Border-image-generator

File: files/en-us/_wikihistory.json [hunk 4]
```diff
@@ -159936,5 +159927,14 @@
       "Sheppy",
       "jswisher"
     ]
+  },
+  "web/css/css_backgrounds_and_borders/border-image_generator": {
```
Comment (teoli2003) [inline files/en-us/_wikihistory.json:159931]: Capitalization is incorrect

File: files/en-us/_wikihistory.json [hunk 5]
```diff
@@ -101430,6 +101398,29 @@
...
+    "modified": "2019-03-20T10:29:48.350Z",
+    "contributors": [
+      "alattalatta",
+      "mfluehr",
+      "Sheppy",
+      "teoli",
+      "gabriel_ivanica"
+    ]
+  },
+  "Web/CSS/CSS_Backgrounds_and_Borders/box-shadow_generator": {
```
Comment (teoli2003) [inline files/en-us/_wikihistory.json:101411]: Missing a capital

File: files/en-us/_redirects.txt [hunk 6]
```diff
@@ -10030,9 +10033,9 @@
 /en-US/docs/Web/CSS/Selector_lists	/en-US/docs/Web/CSS/Selector_list
 /en-US/docs/Web/CSS/Selectors	/en-US/docs/Web/CSS/CSS_Selectors
 /en-US/docs/Web/CSS/Text-overflow_TEST	/en-US/docs/Web/CSS/text-overflow
-/en-US/docs/Web/CSS/Tools/Border-image_generator	/en-US/docs/Web/CSS/CSS_Background_and_Borders/Border-image_generator
-/en-US/docs/Web/CSS/Tools/Border-radius_generator	/en-US/docs/Web/CSS/CSS_Background_and_Borders/Border-radius_generator
-/en-US/docs/Web/CSS/Tools/Box-shadow_generator	/en-US/docs/Web/CSS/CSS_Background_and_Borders/Box-shadow_generator
+/en-US/docs/Web/CSS/Tools/Border-image_generator	/en-US/docs/web/css/css_backgrounds_and_borders/border-image_generator
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:10036]: Capitals!

File: files/en-us/_redirects.txt [hunk 7]
```diff
@@ -10030,9 +10033,9 @@
...
+/en-US/docs/Web/CSS/Tools/Border-radius_generator	/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/border-radius_generator
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:10037]: Missing a capital letter

File: files/en-us/_redirects.txt [hunk 8]
```diff
@@ -10030,9 +10033,9 @@
...
+/en-US/docs/Web/CSS/Tools/Box-shadow_generator	/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/box-shadow_generator
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:10038]: Missing a capital letter

File: files/en-us/_redirects.txt [hunk 9]
```diff
@@ -9967,10 +9967,13 @@
 /en-US/docs/Web/CSS/CSS3_Columns	/en-US/docs/Web/CSS/CSS_Columns/Using_multi-column_layouts
 /en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations/Detecting_CSS_animation_support	/en-US/docs/Web/CSS/CSS_Animations/Detecting_CSS_animation_support
 /en-US/docs/Web/CSS/CSS_Background_and_Borders	/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders
+/en-US/docs/Web/CSS/CSS_Background_and_Borders/Border-image_generator	/en-US/docs/web/css/css_backgrounds_and_borders/border-image_generator
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:9970]: Capitalization is incorrect

File: files/en-us/_redirects.txt [hunk 10]
```diff
@@ -9967,10 +9967,13 @@
...
+/en-US/docs/Web/CSS/CSS_Background_and_Borders/Border-radius_generator	/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/border-radius_generator
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:9971]: Missing a capital letter

File: files/en-us/_redirects.txt [hunk 11]
```diff
@@ -9967,10 +9967,13 @@
...
+/en-US/docs/Web/CSS/CSS_Background_and_Borders/Box-shadow_generator	/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/box-shadow_generator
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:9972]: Missing a capital letter

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Capitalization is inconsistent and incorrect

File: files/en-us/_redirects.txt (hunk 11 above)
Comment (AnilSeervi) [reply files/en-us/_redirects.txt:9972]: I've opened a new PR with proper capitalization #8121

**Comment by AnilSeervi** [comment]
Opened #8121 with proper capitalization
Closing this now.

**Pull request closed without merging**